samples/matter/template/README.rst

```diff
@@ -115,35 +115,28 @@ Device Firmware Upgrade support
-Currently, there is no support for utilizing external flash memory for this purpose.
+Similarly to other supported platforms, the Device Firmware Upgrade (DFU) for the nRF54L15 PDK is supported in a configuration that stores the upgraded image in the external FLASH memory.
+The support for Matter OTA DFU with this configuration is enabled by default for both ``debug`` and ``release`` nRF54L15 PDK build variants.
 
-To build the sample with DFU support, use the ``-DCONF_FILE=prj_release.conf`` flag in your CMake build command.
-
-The following is an example command to build the sample with support for OTA DFU only:
+If you want to build the sample with support for both Matter OTA DFU and Bluetooth® SMP, use the following command:
 
 .. code-block:: console
 
-    west build -b nrf54l15pdk_nrf54l15_cpuapp -- -DCONF_FILE=prj_release.conf
-
-If you want to build the sample with support for both OTA DFU and SMP DFU, use the following command:
-
-.. code-block:: console
+    west build -b nrf54l15pdk_nrf54l15_cpuapp -- -DCONFIG_CHIP_DFU_OVER_BT_SMP=y
 
-    west build -b nrf54l15pdk_nrf54l15_cpuapp -- -DCONF_FILE=prj_release.conf -DCONFIG_CHIP_DFU_OVER_BT_SMP=y
+.. matter_template_nrf54l15_build_with_dfu_end
 
-You can disable DFU support for the ``release`` build configuration to double available application memory space.
-Do this by setting the :kconfig:option:`CONFIG_CHIP_DFU_OVER_BT_SMP` and :kconfig:option:`CONFIG_CHIP_OTA_REQUESTOR` Kconfig options to ``n``, and removing the :file:`pm_static_nrf54l15pdk_nrf54l15_cpuapp_release.yml` file.
+Additionally, in this sample the Device Firmware Upgrade (DFU) for the nRF54L15 PDK can also be configured to use the internal MRAM for storage solely.
+This means that both the currently running firmware and the new firmware to be updated will be stored within the device's internal flash memory.
+This configuration is only available for the ``release`` build variant.
 
-For example:
+The following is an example command to build the sample with support for Matter OTA DFU and DFU over Bluetooth® SMP using internal MRAM only:
 
 .. code-block:: console
 
-    west build -b nrf54l15pdk_nrf54l15_cpuapp -- -DCONF_FILE=prj_release.conf -DCONFIG_CHIP_DFU_OVER_BT_SMP=n -DCONFIG_CHIP_OTA_REQUESTOR=n
-
+    west build -b nrf54l15pdk_nrf54l15_cpuapp -- -DCONF_FILE=prj_release.conf -DCONFIG_CHIP_DFU_OVER_BT_SMP=y -DPM_STATIC_YML_FILE=pm_static_nrf54l15pdk_nrf54l15_cpuapp_internal.yml -Dmcuboot_EXTRA_CONF_FILE=<absolute_path_to_the_template_sample>/child_image/mcuboot/boards/nrf54l15pdk_nrf54l15_cpuapp_release_internal.conf -Dmcuboot_EXTRA_DTC_OVERLAY_FILE=<absolute_path_to_the_template_sample>/child_image/mcuboot/boards/nrf54l15pdk_nrf54l15_cpuapp_release_internal.overlay
 
-.. matter_template_nrf54l15_build_with_dfu_end
+Note that in this case, the size of the application partition is 2 times smaller than when using a configuration with external FLASH memory support.
```

Comment:
```suggestion
Note that in this case, the size of the application partition is half of what it would be when using a configuration with external flash memory support.
```